docs/running_pipelines/management.rst

```diff
@@ -272,18 +272,6 @@ for new jobs that it can run, and will claim and run them automatically
 the case where new workers need to be created that the workflow would
 be waiting for another beekeeper loop.
 
-If you provide the ``-can_respecialize 1`` option to the beekeeper, this
-will allow workers to respecialize. That means a worker, in some cases,
-can claim and work on jobs of a different analysis when needed. If you
-run with ``-can_respecialize 1``, you may notice a significant number of
-workers used throughout the pipeline as long as different analyses share
-the same resource classes (i.e. the resource classes are not too specific).
-For instance, if all the analyses are linked to the same resource-class, a
-single worker in ``-can_respecialize 1`` mode would be able to run the
-whole pipeline !
-Since fewer workers are submitted, the general responsiveness of the
-compute cluster and the pipeline are also increased.
-
```

Comment:
Do we explain somewhere the life cycle of a worker ? That would need to mention that by default, _beekeeper_ submits workers that can respecialize (and what respecialization means)